core/app/models/spree/app_configuration.rb

```diff
@@ -274,6 +274,14 @@ class AppConfiguration < Preferences::Configuration
     #   @return [Boolean] Indicates if stock management can be restricted by location
     preference :can_restrict_stock_management, :boolean, default: false
 
+    # @!attribute [rw] i18n_generic_plural
+    #   This value is used as a count for the pluralization helpers related to I18n
+    #   ex: Spree::Order.model_name.human(count: Spree::I18N_GENERIC_PLURAL
+    #   Related to Solidus issue #1164, this is needed to avoid problems with
+    #   different pluralization calculators
+    #   @return [Integer] default: 2.1
+    preference :i18n_generic_plural, :integer, default: 2.1
```

Comment:
Is there a reason for this to be a preference? I think having it at `Spree::I18N_GENERIC_PLURAL` would probably be sufficient for everyone.